django_fido/views.py

```diff
@@ -263,7 +276,8 @@ def create_fido2_request(self) -> Tuple[Dict, Dict]:
         assert user and user.is_authenticated, "User must not be anonymous for FIDO 2 requests."
         credentials = self.get_credentials(user)
         if not credentials:
-            raise ValueError("Can't create FIDO 2 authentication request, no authenticators.")
+            raise Fido2Error("Can't create FIDO 2 authentication request, no authenticators.",
+                             error_code='NoAuthenticatorsError')
```

Comment:
Error codes should be defined in an enum.

Reply:
Fixed.